src/cli/task-generate.ts

```diff
@@ -123,22 +119,16 @@ const chooseFilesToGenerate = async (): Promise<ReadonlyArray<GenerableExtension
  * The filepath for a given file depends on the path, the user-supplied
  * component name, the extension, and whether we're inside of a test directory.
  *
- * @param coreCompiler  the compiler we're using, here to acces the `.path` module
- * @param path          path to where we're going to generate the component
+ * @param filePath      path to where we're going to generate the component
  * @param componentName the user-supplied name for the generated component
  * @param extension     the file extension
  * @returns the full filepath to the component (with a possible `test` directory
  * added)
  */
-const getFilepathForFile = (
-  coreCompiler: CoreCompiler,
-  path: string,
-  componentName: string,
-  extension: GenerableExtension
-): string =>
+const getFilepathForFile = (filePath: string, componentName: string, extension: GenerableExtension): string =>
   isTest(extension)
-    ? coreCompiler.path.join(path, 'test', `${componentName}.${extension}`)
-    : coreCompiler.path.join(path, `${componentName}.${extension}`);
+    ? join(filePath, 'test', `${componentName}.${extension}`)
```

Comment:
Did the `coreCompiler.path` previously normalize these paths? Do we rely on `Node#join` ([docs](https://nodejs.org/api/path.html#pathjoinpaths)) to do the normalization for us? Just trying to understand what the new rules are going to be for normalization are for us, which IIUC, is that anything written from a config/user-provided string is something we'll wanna normalize

Reply:
yeah it would have normalized everything because it would call the polyfill where everything was wrapped with `normalize` - I'll add in a call here